scripts/apply-version.mjs

```diff
@@ -4,6 +4,6 @@ import { dirname, join, resolve } from "path";
 import { fileURLToPath } from "url";
 
 const basedir = resolve(dirname(fileURLToPath(import.meta.url)), "..");
-const version = process.env.PKG_VERSION || readPkgInfoVersion(join(basedir, "server/package.json"));
+const version = process.env.PKG_VERSION || readPkgInfoVersion(join(basedir, "server", "package.json"));
 
-execSync(`cargo set-version ${version}`, { cwd: "server" });
+execSync(`cargo set-version ${version}`, { cwd: join(basedir, "server", "package.json") });
```

Comment:
`cargo set-version` needs to run in the server directory, not against the package json, though maybe we simply wanna create a script that does this so that we don't have another rust dependency to install in the CI